iron-overlay-behavior.html

```diff
@@ -308,28 +312,25 @@
 
       this._manager.addOrRemoveOverlay(this);
 
+      // TODO handle case when it was already animating, e.g. quick open/close.
```

Comment:
I'll remove this TODO as I think it would be over engineering, & without a real use case it would make this behavior just slower for nothing.